.github/workflows/build.yml

```diff
@@ -0,0 +1,27 @@
+name: Build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK
+        uses: olafurpg/setup-scala@v2
+      - name: Cache Coursier
+        uses: actions/cache@v1
+        with:
+          path: ~/.cache/coursier
+          key: sbt-coursier-cache
```

Comment:
Hi @tpolecat I think you should also add an `${{ hashFiles('**/build.sbt') }}` otherwise new dependencies after this PR will always be pulled.

Reply:
Thank you! I will add that.

![image](https://user-images.githubusercontent.com/1200131/80226516-164e8780-8612-11ea-9c4e-354430ae438b.png)